### PR TITLE
[test] Update `openacc_cuda_mpi_cppstd` check

### PR DESCRIPTION
**Pull request by ajocksch**
segfault

https://jira.cscs.ch/browse/UES-1640

### CODE REVIEW

File: cscs-checks/mch/openacc_cuda_mpi_cppstd.py [hunk 1]
```diff
@@ -46,9 +45,20 @@ def __init__(self):
...
         if self.current_environ.name.startswith('PrgEnv-cray'):
             self.build_system.fflags += ['-hacc', '-hnoomp']
 
+        elif self.current_environ.name.startswith('PrgEnv-nvidia'):
+            self.build_system.fflags += ['-acc']
+            self.build_system.fflags += ['-ta:tesla:cc60']
+            self.build_system.ldflags = [
+                '-acc', '-ta:tesla:cc60', '-Mnorpath', '-lstdc++',
+                '-L/opt/nvidia/hpc_sdk/Linux_x86_64/21.3/cuda/11.2/lib64',
+                '-lcudart'
```
Comment (jjotero) [inline cscs-checks/mch/openacc_cuda_mpi_cppstd.py:59]: I think you should just do `-Mcuda` in here.

Comment (ajocksch) [reply cscs-checks/mch/openacc_cuda_mpi_cppstd.py:59]: you are right, unfortunately the segfault remains

Comment (ajocksch) [reply cscs-checks/mch/openacc_cuda_mpi_cppstd.py]: looks more elegant, although I thought -Mcuda enables mainly cuda-Fortran

Comment (jjotero) [reply cscs-checks/mch/openacc_cuda_mpi_cppstd.py]: Looks like they've changed it to just `-cuda` these days.

**Comment by jjotero** [comment]
@jenkins-cscs retry dom

**Review by jjotero — APPROVED** [review]

**Pull request merged**